granulate_utils/linux/cgroups/cpu_controller.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from abc import abstractmethod
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+from psutil import Process
+
+from granulate_utils.linux.cgroups.base_controller import BaseController
+from granulate_utils.linux.cgroups.cgroup import CgroupCore, ControllerType
+
+
+@dataclass
+class CpuLimitParams:
+    period: int
+    quota: int
+
+
+class CpuController(BaseController):
```

Comment:
it looks to me like we have methods to set cpu limit quota but no methods to set the cpu limit period. and i understand that in v2 both must always be set together. no reason to deprive ourselves of setting the period as well, the code should be simpler that way.

also, instead of setting each variable individually and creating a "special" case for v2, do the opposite; require setting always both with one function call, and "special" case v1.